nestkernel/nest_time.h

```diff
@@ -537,6 +562,9 @@ class Time
   {
     return ld_round( ms * Range::STEPS_PER_MS );
   }
+
+private:
+  unsigned long steps_;
```

Comment:
Otherwise, if I were going complicated, I wouldn't precompute steps, but cache the value computed in calculate_steps, which would only be computed for get_steps, with an invalid marker in steps_ to tell me when I needed to compute. If the caching is worth it, that's a much more efficient way to do so. 

But I think there's probably places in the code where time is being repeatedly converted to steps (like in event), and simply converting and storing the steps is what you really want.